src/main.rs

```diff
@@ -59,71 +58,33 @@ fn main() {
                 // Can this ever happen?
                 Ok(())
             } else {
-                if let Some(ref mut terminal) = term_stdout {
-                    let _ = terminal.fg(term::color::YELLOW);
-                    println!("Fuzzing found errors!");
-                    let _ = terminal.reset();
-                } else {
-                    println!("Fuzzing found errors!");
-                }
+                utils::print_message("Fuzzing found errors!", term::color::YELLOW);
                 process::exit(-1)
             }
         } else {
             result.map(|_| ())
         }
     } else if args.flag_list {
-        list_fuzz_targets(&mut term_stdout)
+        list_fuzz_targets()
             .map(|_| ())
     } else {
-        if let Some(ref mut terminal) = term_stderr {
-            let _ = terminal.attr(term::Attr::Bold);
-            let _ = terminal.fg(term::color::RED);
-            write!(io::stderr(), "Error:")
-                .expect("failed writing to stderr");
-            let _ = terminal.fg(term::color::WHITE);
-            writeln!(io::stderr(), " Invalid arguments. Usage:\n{}", USAGE)
-                .expect("failed writing to stderr");
-            let _ = terminal.reset();
-        } else {
-            writeln!(io::stderr(), "Invalid arguments. Usage:\n{}", USAGE)
-                .expect("failed writing to stderr");
-        }
-
+        utils::write_to_stderr(format!("{}{}", "Invalid arguments. Usage:\n{}", USAGE).as_str());
```

Comment:
This won't do the right thing. You want `format!("Invalid arguments. Usage:\n{}", USAGE)`.